bitnami/rabbitmq/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/rabbitmq
   - https://www.rabbitmq.com
-version: 11.4.0
+version: 11.4.1
```

Comment:
Since this is adding a new feature, we should be increasing the minor (e.g. to 11.5.0).

Reply:
fixed